### ISSUE TITLE
YCM's FindPython module requests CMake 3.7

### ISSUE THREAD

**Issue by rsantos88** [roboticslab-uc3m/yarp-devices#212]
```bash
cmake ..
-- Found YCM: /usr/local/share/YCM/cmake (found version "0.10.2.9-20190508+gitfb91e61")
-- Checking for module 'aravis-0.4'
--   No package 'aravis-0.4' found
CMake Error at /usr/local/share/YCM/cmake-next/Modules/FindPython/Support.cmake:8 (cmake_policy):
  An attempt was made to set the policy version of CMake to "3.7" which is
  greater than this version of CMake.  This is not allowed because the
  greater version may have new policies not known to this CMake.  You may
  need a newer CMake version to build this project.
Call Stack (most recent call first):
  /usr/local/share/YCM/cmake-next/Modules/FindPython.cmake:152 (include)
  CMakeLists.txt:61 (find_package)


-- Setting up installation of roboticslab-yarp-devices.ini to /usr/local/share/yarp/config/path.d folder.
--  --- plugin AmorControlboard: disabled (dependencies unsatisfied)
--  --- plugin AravisGigE: disabled (dependencies unsatisfied)
--  +++ plugin CanBusControlboard: enabled
--  +++ plugin CanBusFake: enabled
--  +++ plugin CanBusHico: enabled
--  --- plugin CanBusPeak: disabled (dependencies unsatisfied)
--  +++ plugin CuiAbsolute: enabled
--  +++ plugin DextraControlboardUSB: enabled
--  +++ plugin FakeControlboard: enabled
--  +++ plugin FakeJoint: enabled
--  +++ plugin Jr3: enabled
--  +++ plugin LacqueyFetch: enabled
--  --- plugin LeapMotionSensor: disabled (dependencies unsatisfied)
--  --- plugin PhidgetSpatial: disabled (dependencies unsatisfied)
--  +++ plugin ProximitySensorsClient: enabled
--  --- plugin SpaceNavigator: disabled (dependencies unsatisfied)
--  +++ plugin TechnosoftIpos: enabled
--  +++ plugin TextilesHand: enabled
--  --- plugin WiimoteSensor: disabled (dependencies unsatisfied)
-- Configuring incomplete, errors occurred!
See also "/home/teo/repos/yarp-devices/build/CMakeFiles/CMakeOutput.log".
See also "/home/teo/repos/yarp-devices/build/CMakeFiles/CMakeError.log".
```

**Comment by PeterBowman** [comment]
This issue was reported and fixed in YCM v0.10.2 (https://github.com/robotology/ycm/issues/248), which is your current version (`found version "0.10.2.9-20190508+gitfb91e61"`). The `/usr/local/share/YCM/cmake-next/Modules/FindPython/Support.cmake` file should be no longer installed, which I presume is a leftover of a previous installation of YCM on your system that shadows the proper `Support.cmake` located at `cmake-3.12/Modules/`. Please `sudo rm -rf /usr/local/share/YCM` and install YCM again.

**Comment by rsantos88** [comment]
Perfect! Thanks @PeterBowman !!
